ppr-api/src/ppr_api/resources/financing_statements.py

```diff
@@ -559,25 +565,33 @@ def post(registration_num):
             if not statement.validate_debtor_name(request_json['debtorName'], is_staff(jwt)):
                 return resource_utils.base_debtor_invalid_response()
 
-            # No fee for a discharge.
+            # No fee for a discharge but create a payment transaction record.
+            # Set up the registration, pay, and save the data.
+            registration = pay_and_save(request_json,
+                                        model_utils.REG_CLASS_DISCHARGE,
+                                        statement,
+                                        registration_num,
+                                        account_id)
             # Try to save the discharge statement: failure throws a business exception.
-            registration = Registration.create_from_json(request_json,
-                                                         model_utils.REG_CLASS_DISCHARGE,
-                                                         statement,
-                                                         registration_num,
-                                                         account_id)
-            registration.save()
-
+            # registration = Registration.create_from_json(request_json,
```

Comment:
remove comment or are you planning to use this later?

Reply:
Removed in latest commit..